src/plugins/columnSorting/columnSorting.js

```diff
@@ -35,19 +43,45 @@ Hooks.getSingleton().register('afterColumnSort');
  * // as a object with initial order (sort ascending column at index 2)
  * columnSorting: {
  *  column: 2,
- *  sortOrder: true, // true = ascending, false = descending, undefined = original order
+ *  sortOrder: 'asc', // 'asc' = ascending, 'desc' = descending, 'none' = original order
  *  sortEmptyCells: true // true = the table sorts empty cells, false = the table moves all empty cells to the end of the table
  * }
  * ...
  * ```
- * @dependencies ObserveChanges
+ * @dependencies ObserveChanges moment
  */
 class ColumnSorting extends BasePlugin {
-
   constructor(hotInstance) {
     super(hotInstance);
-    this.sortIndicators = [];
+    this.sortIndicators = []; // TODO: It could be refactored, we are sorting just by one column, without saving state of previous sort indicators.
+    /**
+     * Visual index of last sorted column.
+     *
+     * @type {null|Number}
+     */
     this.lastSortedColumn = null;
+    /**
+     * Visual index of sorted column.
+     */
+    this.sortColumn = void 0;
+    /**
+     * Order of sorting. For `asc` ascending order, for `desc` descending order, for `none` the original order.
+     *
+     * @type {String}
+     */
+    this.sortOrder = NONE_SORT_STATE;
+    /**
+     * Object containing visual row indexes mapped to data source indexes.
+     *
+     * @type {RowsMapper}
+     */
+    this.rowsMapper = new RowsMapper(this);
+    this.sortingEnabled = false; // TODO: It should be removed, when global ArrayMapper will be used.
```

Comment:
Maybe we can give this variable a miss because this is more as a workaround, not a true solution.
Otherwise, please add a description of the variable, above the definition.